tools/scripts/aducm.mk

```diff
@@ -223,9 +223,35 @@ $(MQTT_LIB):
 
 endif
 
+#	IIO
+IIO_DIR = $(NO-OS)/libraries/iio
+#If mqtt is found in SRC_DIRS
+ifneq ($(if $(findstring $(IIO_DIR), $(SRC_DIRS)), 1),)
+#Remove iio from srcdirs because it is a library
+SRC_DIRS := $(filter-out $(IIO_DIR), $(SRC_DIRS))
+
+IIO_LIB		= $(IIO_DIR)/libiio.a
+
+LIBS_DIRS	+= "$(IIO_DIR)"
+LIBS		+= $(notdir $(IIO_DIR))
+INCLUDE_DIRS	+= $(IIO_DIR)
```

Comment:
LIBS_DIRS has quotes. Is it for whitespaces in paths ? Then perhaps INCLUDE_DIRS should also have quotes.

Reply:
The quotes were needed to link the libraries to a project.
In this PR I will keep this way for consistency, but will fix them in an other PR with suggestions from Andrei too